test/export/test_converter.py

```diff
@@ -17,45 +17,62 @@
 
 class TestConverter(TestCase):
     def _check_equal_ts_ep_converter(
-        self, mod, inp, option: Union[List[str]] = None
+        self,
+        M,
+        inp,
+        option: Union[List[str]] = None,
+        check_persistent=False,
     ) -> ExportedProgram:
         # By default, it tests both jit.trace and jit.script.
         if option is None:
             option = ["trace", "script"]
 
-        model_list = []
+        if check_persistent:
+            num_iterations = 10
+        else:
+            num_iterations = 1
+
+        ep_list = []
         for opt in option:
             if opt == "script":
-                ts_model = torch.jit.script(mod)
+                # Separate two models for testing non-functional effects
+                if check_persistent:
+                    original_ts_model = torch.jit.script(M())
+                    ts_model = torch.jit.script(M())
+                else:
+                    original_ts_model = torch.jit.script(M)
+                    ts_model = torch.jit.script(M)
+            elif opt == "trace":
+                if check_persistent:
+                    original_ts_model = torch.jit.trace(M(), inp)
+                    ts_model = torch.jit.trace(M(), inp)
+                else:
+                    original_ts_model = torch.jit.trace(M, inp)
+                    ts_model = torch.jit.trace(M, inp)
             else:
-                ts_model = torch.jit.trace(mod, inp)
-            model_list.append(ts_model)
+                raise RuntimeError(f"Unrecognized mode for torch.jit: {opt}")
 
-        ep_list = []
-        for ts_model in model_list:
-            ep = TS2EPConverter(ts_model, inp).convert()
-            ep_list.append(ep)
-            ep_out, _ = pytree.tree_flatten(ep.module()(*inp))
-            orig_out, _ = pytree.tree_flatten(ts_model(*inp))
-
-            # Check module.
-            if isinstance(mod, torch.nn.Module):
-                self.assertEqual(
-                    ep.state_dict.keys(),
-                    ts_model.state_dict().keys(),
-                )
+            for _ in range(num_iterations):
+                orig_out, _ = pytree.tree_flatten(original_ts_model(*inp))
 
-            # Check results.
-            self.assertEqual(len(ep_out), len(orig_out))
-            for ep_t, orig_t in zip(ep_out, orig_out):
-                if isinstance(ep_t, torch.Tensor) and isinstance(orig_t, torch.Tensor):
-                    self.assertEqual(ep_t.shape, orig_t.shape)
-                    self.assertTrue(torch.allclose(ep_t, orig_t))
-                else:
-                    self.assertEqual(type(ep_t), type(orig_t))
-                    self.assertEqual(ep_t, orig_t)
+                ep = TS2EPConverter(ts_model, inp).convert()
```

Comment:
Just for my knowledge, why do we need to to convert it for number of iterations?

If it is for checking persistence, should we convert it once and run the module for number of iterations?